File: ComplementApp.API/Models/ActividadGeneral.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ComplementApp.API.Models
{
    [Table("TActividadGeneral")]
    public class ActividadGeneral
    {
        public int ActividadGeneralId { get; set; }        

        [Column(TypeName = "decimal(30,8)")]
        public decimal ApropiacionVigente { get; set; }

        [Column(TypeName = "decimal(30,8)")]
        public decimal ApropiacionDisponible { get; set; }    
        public int?  RubroPresupuestalId { get; set; }
        public RubroPresupuestal RubroPresupuestal { get; set; }
        public int? PciId { get; set; }
        public Pci Pci { get; set; }
    }
}

File: ComplementApp-SPA/src/app/_models/actividadGeneral.ts
import { ValorSeleccion } from '../_dto/valorSeleccion';
import { RubroPresupuestal } from './rubroPresupuestal';

export class ActividadGeneral {
  actividadGeneralId: number;
  apropiacionVigente: number;
  apropiacionDisponible: number;
  rubroPresupuestal: RubroPresupuestal;
  situacionFondo: ValorSeleccion;
  fuenteFinanciacion: ValorSeleccion;
  recursoPresupuestal: ValorSeleccion;
}
